### DISCUSSION TITLE
Not seeing span events for logs when using logback appender

### DISCUSSION THREAD

**Discussion by guadalupe-duolingo** [open-telemetry/opentelemetry-java-instrumentation#9082] [Q&A]
I'm not sure if I'm doing something wrong here but I'm trying to use [the logback appender ](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/logback/logback-appender-1.0/library) and I'm initializing it like this in my `main` but I'm not seeing any span events when an error is logged:
```
OpenTelemetryAppender.install(OpenTelemetrySdk.builder().build());
```
I have this in my `build.gradle` file:
```
implementation 'io.opentelemetry:opentelemetry-sdk:1.28.0'
implementation group: "io.opentelemetry.instrumentation", name: "opentelemetry-logback-appender-1.0", version: '1.28.0alpha'
```
Any suggestions on what I can try to get this to work? Thanks.

**Comment by laurit** [comment]
The logback appender you referenced sends log events to configured destination. It does not create span events when errors are logged.
